src/sentry/conf/server.py

```diff
@@ -1114,20 +1072,10 @@ def create_partitioned_queues(name):
     "organizations:sso-saml2": True,
     # Enable workaround for migrating IdP instances
     "organizations:sso-migration": False,
-    # Return unhandled information on the issue level
-    "organizations:unhandled-issue-flag": True,
     # Enable percent-based conditions on issue rules
     "organizations:issue-percent-filters": True,
     # Enable the new images loaded design and features
     "organizations:images-loaded-v2": True,
-    # Enable the mobile screenshots feature
-    "organizations:mobile-screenshots": False,
-    # Store release bundles as zip files instead of single files
-    "organizations:release-archives": False,
-    # Enable the release details performance section
-    "organizations:release-comparison-performance": False,
-    # Enable percent displays in issue stream
-    "organizations:issue-percent-display": False,
```

Comment:
i think this feature still exists

Reply:
If missed this one. I'll double check all the removals for the unprefixed names too.